File: ncDados/dCaracteristicaItem.vb
Namespace nsCaracteristica

  Public Class ColecaoCaracteristicaItem
    Inherits List(Of dCaracteristicaItem)
  End Class

  Public Class dCaracteristicaItem

    Private _cid As Nullable(Of Integer)
    Private _caracteristicas_cid As Nullable(Of Integer)
    Private _valor As String

    Public Property cid() As Nullable(Of Integer)
      Get
        Return _cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cid = value
      End Set
    End Property

    Public Property caracteristicas_cid() As Nullable(Of Integer)
      Get
        Return _caracteristicas_cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _caracteristicas_cid = value
      End Set
    End Property

    Public Property valor() As String
      Get
        Return _valor
      End Get
      Set(ByVal value As String)
        _valor = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dCaracteristicaItem.cs
using System;
using System.Collections.Generic;

namespace nsCaracteristica
{
    public class ColecaoCaracteristicaItem : List<dCaracteristicaItem>
    {
    }

    public class dCaracteristicaItem
    {
        private int? _cid;
        private int? _caracteristicas_cid;
        private string _valor;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public int? caracteristicas_cid
        {
            get => _caracteristicas_cid;
            set => _caracteristicas_cid = value;
        }

        public string valor
        {
            get => _valor;
            set => _valor = value;
        }
    }
}
